app/Http/Controllers/API/TraderController.php

```diff
@@ -183,7 +147,8 @@ public function emailVoucherHistory(Request $request, Trader $trader)
         // If a date is provided generate a specific response message.
         if ($date) {
             $response_text = trans(
-                'api.messages.email_voucher_history_date', [
+                'api.messages.email_voucher_history_date',
+                [
```

Comment:
Not sure about some of these syntax changes - but I know it's your editor. No harm done, but we might want to think about normalising a little.